packages/did-auth-siop-op-authenticator/agent.yml

```diff
@@ -0,0 +1,107 @@
+version: 3.0
+
+constants:
+  baseUrl: http://localhost:3335
+  port: 3335
+  # please use your own X25519 key, this is only an example
+  secretKey: 29739248cad1bd1a0fc4d9b75cd4d2990de535baf5caadfdf8d8f86664aa830c
+  methods:
+    - resolveDid
```

Comment:
Are these all methods? Why do we expose a resolveDid, whilst we also rely on the DID plugin already, which exposes something similar?

Reply:
wasnt needed, removed